frontend/benefit/applicant/src/components/applications/forms/application/styled.ts

```diff
@@ -94,6 +94,10 @@ const StyledSubSection = styled.div`
   }
 `;
 
+const StyledIBANField = styled(TextInput)<TextInputProps>`
+  min-width: 14em;
```

Comment:
Still not perfect... it might be hard to put the masking into hds-component?
<img width="441" alt="Screenshot 2021-06-18 at 10 01 20 AM" src="https://user-images.githubusercontent.com/7648194/122520695-9dcc3680-d01c-11eb-8643-f4cefcc5b9ce.png">

Reply:
Passing the `size` attribute to the input did the trick 😉

Reply:
It still overflows with "FI 88 888..."  when I tried it here: https://helsinkibenefit-bf-appl-98.test.kuva.hel.ninja